### ISSUE TITLE
2 Kia's, 2 UVO Account Issues.

### ISSUE THREAD

**Issue by sbcrumb** [Hyundai-Kia-Connect/kia_uvo#805]
Please check Services, Known Bug / Issues and Troubleshooting over here first: https://github.com/fuatakgun/kia_uvo/blob/master/README.md
**Region and Brand of car**

**Version of the integration**
2.18.2

**Describe the bug**
My wife and I both have KIA's each of us have our OWN UVO account. Each is the Secondary drive of the other.
In the app/integration, I have no way to specify which UVO account send the command.
So as an example. if my wife starts her car(through HA) it will send as me the secondary driver of that car. The car profile will be mine, He phone won't connect to bluetooth etc. Is there a way we can specify which account to send the data from to prevent this?

Right now my only work around is to remove us each as secondary drivers so only primary is shown.
(Which I don't like cause when I want to use the KIA App I can't see her car unless I log out and back in)

**Comment by cdnninja** [comment]
Just to confirm you have two instances of the integration installed.   So two copies of each car.  

You want to be able to select which login to use to send the command?

**Comment by sbcrumb** [comment]
Yes each user has 2 Device. However cause they are named The same they don't appear as separate devices(cars)
And Yes I want to say send this command as this UVO login or that UVO login. So that it will send it as the primary or secondary driver.. I hope this makes sense.

**Comment by cdnninja** [comment]
It does.   Definitely not a use case we thought about when we did this.  

We dealt with 2 cars to one account in 2.x of this. It also handles discrete accounts but not that combo.  

I'll poke through the code to see how we would do this.

**Comment by sbcrumb** [comment]
Thanks for now, I just removed each of us as secondary accounts. So when I send commands to her car it is as her, and mine it is as me and nothing in between for now. Let me know if I can help out at all.
